project_generation/manifest.go

```diff
@@ -289,12 +302,36 @@ var apiFiles = []fileGen{
 		extension:    ".go",
 		filePrefix:   "",
 	},
+	{
+		templatePath: "api/features/helloworld.feature",
+		outputPath:   "features/helloworld",
+		extension:    ".feature",
+		filePrefix:   "",
+	},
+	{
+		templatePath: "api/features/steps/example_component.go.tmpl",
+		outputPath:   "features/steps/example_component",
+		extension:    ".go",
+		filePrefix:   "",
+	},
+	{
+		templatePath: "api/features/steps/steps.go.tmpl",
```

Comment:
the `.tmpl` extension is appended when the struct is used, can you remove it please?